presto-product-tests/conf/docker/common/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+services:
+  hadoop-master-common:
+    hostname: hadoop-master
+    image: 'teradatalabs/cdh5-hive:2'
+    ports:
+      - '1080:1080'
+      - '8020:8020'
+      - '8088:8088'
+      - '9083:9083'
+      - '10000:10000'
+      - '50070:50070'
+      - '50075:50075'
+
+  hadoop-master-kerberos:
+    extends:
+      service: hadoop-master-common
+    image: 'teradatalabs/cdh5-hive-kerberized:2'    
```

Comment:
@brian-rickman : The difference between :2 is :3 is the java version. tag 3 has the new java version 1.8u92